.nvmrc

```diff
@@ -0,0 +1 @@
+.node-version
```

Comment:
This is a symlink to the `.node-version` file. This is for Travis CI but also so we can store our Node version in a single place!

Docs: https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#specifying-nodejs-versions-using-nvmrc